models.py

```diff
@@ -18,6 +23,11 @@ class User(db.Model):
     password = db.Column(db.String, nullable=False)
     family_id = db.Column(db.Integer, ForeignKey('family.family_id'))
```

Comment:
This relationship is required, you should be able to have a user without a family ID.